blocks/library/categories/index.php

```diff
@@ -48,6 +48,10 @@ function render_block_core_categories( $attributes ) {
 
 	$class = "wp-block-categories wp-block-categories-{$type} align{$align}";
 
+	if ( isset( $attributes['className'] ) ) {
```

Comment:
Out of curiosity, how does it work without `attributes` specified on the server? :D

Reply:
Hi @gziolo, I was also curious about this logic while doing the changes. This works because  prepare_attributes_for_render, just returns the attributes parsed from the comment demarcations if the attributes are not "registered" on the server.
https://github.com/WordPress/gutenberg/blob/eee927878b27198bf19388a2d4add77b15cbc510/lib/class-wp-block-type.php#L128

Reply:
Good to know, it needs to be fixed soon (another PR) :)